packages/core/src/constants.ts

```diff
@@ -1,8 +1,10 @@
 export const LAYER_DEFAULT = 'default'
 export const LAYER_PREFLIGHTS = 'preflights'
 export const LAYER_SHORTCUTS = 'shortcuts'
+export const LAYER_IMPORTS = '@imports'
```

Comment:
```suggestion
export const LAYER_IMPORTS = 'imports'
```

Maybe in the context, `imports` are already explicit enough?